antsibull/data/collection-enum.py

```diff
@@ -195,6 +229,13 @@ def main(args):
     for plugin_type in C.DOCUMENTABLE_PLUGINS:
         result['plugins'][plugin_type] = load_all_plugins(plugin_type, basedir, coll_filter)
 
+    # Export role docs
+    RoleMixin = getattr(doc, 'RoleMixin', None)
```

Comment:
Just FYI, neither `RoleMixin` nor any of its methods were ever meant to be publicly accessible. Using this is at-your-own-risk as my intention _was_ to try to make a proper public API for some of this in the future.

Reply:
Actually most code in collection-enum.py is using internal stuff from ansible-doc that can break at any moment. I would prefer to have a stable API, but unfortunately there isn't one :)

Reply:
(In case it breaks, you can configure antsibull to use `ansible-doc --json` to extract all the docs. Unfortunately it's like 1000x slower, that's why this method is the default :) )